src/detections/hooks/use-split.js

```diff
@@ -0,0 +1,7 @@
+import { useLocation } from 'react-router-dom'
+
+export const useSplit = () => {
```

Comment:
Let's rename this to something like `useCameraIds` or `useCameraFilter`

Reply:
Done